docker/Dockerfile.dev

```diff
@@ -0,0 +1,40 @@
+# Copyright Robert Bosch GmbH, 2020. Part of the Eclipse Kuksa Project.
+#
+# All rights reserved. This configuration file is provided to you under the
+# terms and conditions of the Eclipse Distribution License v1.0 which
+# accompanies this distribution, and is available at
+# http://www.eclipse.org/org/documents/edl-v10.php
+# Quick docker for doing development
+
```

Comment:
Should we use EPL 2.0 here? At least the license file in the root directory of this repository and  the other repositories of the Eclipse Kuksa project use EPL 2.0.

Reply:
Hi, thanks for taking a look! 

My understanding was, that for "config files" or "Example" files, and basically everything that is very simple and does not include any logic, we use another Eclipse license (EDL?) , because it is like a BSD license and allows people to just copy it.

Not sure where the discussion was, but here are also EPL and EDL: https://github.com/eclipse/kuksa.apps/pull/28/commits/a63511d25f8ae264c1a52f900347766afb50ec4f

@rohoet Do you remember? Does it make sense like this, or should some licenses be changed?

Reply:
@bs-jokri https://www.eclipse.org/org/documents/edl-v10.php 
> Use of the Eclipse Distribution License by any project at the Eclipse Foundation must be reviewed and unanimously approved by the Board of Directors.

Does that mean we have to approve every EDL related files?

Reply:
@rohoet @SebastianSchildt @eriksven No, I understand the comment at the file as if you plan to release a whole project under EDL you need board approval. The EDL is explicitly made for licensing small files with low level of originality. As I understand those files are exactly that. So EDL seems to be perfectly suitable.

Reply:
Ok, thanks for clarification!